openlcsd/flow/tasks.py

```diff
@@ -1369,8 +1369,9 @@ def fork_imports(context, engine):
                 ],
                 [
                     IF_ELSE(
-                        lambda o, e: 'image' in o.get('build_type'),
-                        # Work flow for import container build
+                        lambda o, e: 'image' in o.get('build_type') and not o.get('parent'),  # noqa
```

Comment:
The bug is a regression of  https://github.com/RedHatProductSecurity/openlcs/pull/83/files#diff-8c97d22701f1907296fe85398652366a19a776988575e3024ff336651e9bea7aR1094, not sure why it was removed.

Reply:
Yes, it is a regression